plugins/com.google.cloud.tools.eclipse.appengine.deploy/plugin.xml

```diff
@@ -25,7 +25,7 @@
   
   <extension point="org.eclipse.ui.handlers">
     <handler
-      class="com.google.cloud.tools.eclipse.appengine.deploy.AppEngineStandardDeployCommandHandler"
+      class="com.google.cloud.tools.eclipse.appengine.deploy.standard.StandardDeployCommandHandler"
       commandId="com.google.cloud.tools.eclipse.appengine.deploy.standard">
```

Comment:
Typically we'd have a single command ('com.google.cloud.tools.eclipse.appengine.deploy') that would have different Standard and Flex handlers that use `<enabledWhen>` expressions to determine where they're applicable.